DuckDuckGo/Favicons.swift

```diff
@@ -392,6 +392,7 @@ public class Favicons {
         return [
             .downloader(Constants.downloader),
             .requestModifier(Constants.requestModifier),
+            .cacheSerializer(FormatIndicatedCacheSerializer.jpeg),
```

Comment:
What about entries that exist in the cache? Is migration possible/needed?

Reply:
Yes, we'll need to do something to handle existing cached images. `cacheSerializer` will convert images that haven't already been converted when reading them from the cache, but that doesn't apply to widgets as we don't go through Kingfisher to read them – currently looking into a way to solve this.

Reply:
Images are now converted to sRGB directly before rendering to avoid this issue. They're small and there will only be a fixed number of them, so it should be okay. Technically doing this negates the need to have this `cacheSerializer`, so it could be removed.